packages/katello/katello/katello.spec

```diff
@@ -104,7 +103,6 @@ Common runtime components of %{name}
 Summary: Katello Debug utilities
 Group: Applications/System
 Requires: bash
-Requires: mktemp
```

Comment:
given we'll be getting rid of it completely soon, is it really worth it?

Reply:
Not sure, are we planning to get rid of it? If so, why make it optional now instead of just get rid of it altogether now. I'm not opposed to merging this now, but I want to consider the bigger picture as well. If we never publish it on EL9, we don't need to deprecate it either.

Reply:
My recollection of the discussion with @ehelms was that we want to drop it altogether, but shall investigate if we really have everything ported to sos properly.

I agree that never have to deprecate/remove it from EL9 is nice, but we have until 3.10 branching to get that part cleared out, while I'd prefer to have sort of working nightlies before that. Making it optional now is super cheap and allows us to untangle those two (otherwise unrelated) work items.

Reply:
(fixed)

Reply:
We can leave it out of el9 comps for now as well. So make it optional but don't drop it yet